errcheck/errcheck.go

```diff
@@ -123,48 +125,50 @@ func (e byName) Less(i, j int) bool {
 	return ei.Line < ej.Line
 }
 
-// Checker checks that you checked errors.
-type Checker struct {
-	// Ignore is a map of package names to regular expressions. Identifiers from a package are
-	// checked against its regular expressions and if any of the expressions match the call
-	// is not checked.
-	Ignore map[string]*regexp.Regexp
-
-	// Blank, if true, means assignments to the blank identifier are also considered to be
-	// ignored errors.
-	Blank bool
-
-	// Asserts causes ignored type assertion results to also be checked.
-	Asserts bool
-
-	// Tags are a list of build tags to use.
-	Tags []string
+// Exclusions define symbols and language elements that will be not checked
+type Exclusions struct {
+	// Packages lists regular expression patterns that exclude whole packages.
+	Packages []string
 
-	// Verbose causes extra information to be output to stdout.
-	Verbose bool
+	// Symbols lists regular expression patterns that exclude package symbols.
```

Comment:
Hold on a second... these are not regular expressions

Reply:
Fixed comment